src/ldf-client-ui.js

```diff
@@ -734,11 +735,10 @@ require('leaflet/dist/images/marker-shadow.png');
             geoFeature.properties.name = valueLabel.split('@', 1)[0].replace(/['"]+/g, '');
 
           // Add feature to map
-          L.geoJSON(geoFeature, {
+          let laayer = L.geoJSON(geoFeature, {
             onEachFeature: function (feature) {
               // Add feature data to map
-              self.mapLayer.addData(feature);
-
+              // self.mapLayer.addData(feature);
```

Comment:
Unneeded comment

Reply:
Yes, sorry about that. we actually need to do away with that `self.mapLayer.addData(feature);` completely in the code. Shouldn't be part.